corehq/apps/hqadmin/service_checks.py

```diff
@@ -94,7 +94,7 @@ def check_elasticsearch():
     if cluster_health == 'red':
         return ServiceStatus(False, "Cluster health at %s" % cluster_health)
 
-    doc = {'_id': 'elasticsearch-service-check-{}'.format(random_hex()[:7]),
+    doc = {'_id': 'elasticsearch-service-check-{}'.format(uuid.uuid4().hex[:7]),
```

Comment:
i guess the chances of having two uuids with the same first 7 characters is pretty remote.

Reply:
should be roughly the same chance as it was before with random_hex